mysql/userstat/userstat.go

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2015 Square, Inc
+//
+
+package userstat
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"reflect"
+	"regexp"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/square/inspect/metrics"
+	"github.com/square/inspect/mysql/tools"
+	"github.com/square/inspect/os/misc"
```

Comment:
`os.misc` is used in `cmd` as well. I think that we should leave it like that. For the `util` package, it would be under `mysql` since it's only for `mysql` and not anything else.